docs/library/stm.rst

```diff
@@ -102,3 +102,52 @@ the second CPU, the RF core.
     Execute a HCI command on the SYS channel.  The execution is synchronous.
 
     Returns a bytes object with the result of the SYS command.
+
+Functions specific to STM32WLxx MCUs
+------------------------------------
+
+These functions are available on STM32WLxx microcontrollers, and interact with
+the integrated "SUBGHZ" radio modem peripheral.
+
+.. function:: subghz_ext_hse32(external_hse32)
+
+    Call this function to configure the HSE32 external oscillator source,
+    based on the ``external_hse32`` parameter:
+
+    - If truthy, an external HSE32 clock signal is provided and the internal HSE
+      should be bypassed.
+    - If falsey, the internal HSE32 oscillator should be enabled. This is the
+      default state when the board boots.
+
+   This function should not be called while the radio peripheral is being used.
+
+.. function:: subghz_cs(deassert)
+
+   Sets the internal SPI CS pin attached to the radio peripheral. The
+   ``deassert`` argument has the same behaviour as calling :ref:`machine.Pin()
+   <machine.Pin>` to set the output: a truthy value means de-asserted, falsey means
+   asserted.
```

Comment:
I think this needs a little more explanation, like "a truthy value means "CS pin high" which deasserts the line.  a falsey value means "CS pin low" which asserts it."